product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/third_party_integrations/datadog.mdx

```diff
@@ -45,18 +45,22 @@ The set of metrics delivered to Datadog is subject to change. Metrics with names
 
 ## Cost
 
-After enabling the BigAnimal telemetry integration, check your billable Datadog usage and continue to monitor it over time.
+Telemetry sent to Datadog will incur charges on your Datadog plan according to the [Datadog price list](https://www.datadoghq.com/pricing/list/).
+We recommend that you continually monitor your billable Datadog usage as the cost could change.
+Datadog provides [usage metrics](https://docs.datadoghq.com/account_management/billing/usage_metrics/) for this purpose, such as `datadog.estimated_usage.hosts`, `datadog.estimated_usage.containers`, and `datadog.estimated_usage.metrics.custom`.
 
 Be aware of the following cost considerations:
 
-* *You are responsible for all costs* charged to your Datadog account by telemetry sent by the BigAnimal Datadog integration. Charges are based on usage but they can also result from BigAnimal errors or oversights. Enable the Datadog integration only if you accept this responsibility.
-* Datadog bills for each monitored Postgres node (including any replicas) as a "monitored host" in the your Datadog plan. See the [Datadog price list](https://www.datadoghq.com/pricing/list/) for current pricing per monitored host.
-* Datadog might bill for each Postgres container and monitoring agent container running on each monitored Postgres node. See the [Datadog price list](https://www.datadoghq.com/pricing/list/) for current pricing per monitored container.
-* Datadog counts some of the metrics sent by BigAnimal as custom metrics. Datadog plans include a limited number of free unique custom metrics dimensions per host. Additional metrics dimensions above the free limit are billable at a rate set in the Datadog price list. See your [Datadog plan price list](https://www.datadoghq.com/pricing/list/) for details. Although BigAnimal's telemetry integration for Datadog tries to limit custom metrics delivery to minimize additional billable Datadog usage, you can still expect some billable custom metrics usage.
+* Datadog bills for each monitored Kubernetes node as a "Monitored Host".
```

Comment:
There is no single "Monitored Host" line item on the Datadog price list (unlike "Container Monitoring" for example, which is easy to find) can we refer to a specific line?